.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Let's keep this enabled. It doesn't do any harm and is sometimes useful.

Reply:
Sounds good!